src/generated/Normalizer/PriceNormalizer.php

```diff
@@ -52,6 +52,16 @@ public function denormalize($data, $class, $format = null, array $context = arra
         elseif (property_exists($data, 'value') && $data->{'value'} === null) {
             $object->setValue(null);
         }
+        if (property_exists($data, 'appliedPromotions') && $data->{'appliedPromotions'} !== null) {
+            $values = array();
```

Comment:
```suggestion
            $values = [];
```

Reply:
This file is automatically generated by jane, I would not change it